src/io/dss/dss2eng.jl

```diff
@@ -547,6 +547,15 @@ end
 
 "Adds transformers to `data_eng` from `data_dss`"
 function _dss2eng_transformer!(data_eng::Dict{String,<:Any}, data_dss::Dict{String,<:Any}, import_all::Bool)
+    # check if regcontrol items are present
+    reg_obj = Dict()
+    if haskey(data_dss, "regcontrol")
+        for (id, dss_obj) in get(data_dss, "regcontrol", Dict{String,Any}())
+            reg_obj["$(dss_obj["transformer"])"] = dss_obj
+            reg_obj["$(dss_obj["transformer"])"]["name"] = id
+        end
+    end
+
```

Comment:
you should still do this in a separate `_dss2eng_regcontrol!`, so that you don't have to iterate over dict values so many times

```julia
function _dss2eng_regcontrol!(data_eng::Dict{String,<:Any}, data_dss::Dict{String,<:Any}, import_all::Bool)
    for (id, dss_obj) in get(data_dss), "regcontrol", Dict{String,Any}())
        # build "controls" dict eng_obj here
        data_eng["transformer"]["$(dss_obj["transformer"])"]["controls"] = eng_obj
    end
end
```